cmd/jaeger/internal/extension/jaegerquery/server_test.go

```diff
@@ -301,6 +302,7 @@ func TestServerAddMetricsStorage(t *testing.T) {
 		{
 			name:           "Metrics storage unset",
 			config:         &Config{},
+			qSvcOpts:       &querysvc.QueryServiceOptions{},
```

Comment:
what's the point of defining these as test case parameter if they don't change across tests?

Reply:
I modelled this based on the archive storage test.